services/spar/src/Spar/Scim.hs

```diff
@@ -148,111 +150,129 @@ validateSCIMUser ScimTokenInfo{stiIdP} user = do
 --
 --   * @displayName@ is mapped to our 'userName'.
 --
---   * A mandatory @SAML.UserRef@ is derived from 'SCIM.User.externalId'
---     and the 'idpId' (retrieved via SCIM token).  FUTUREWORK: We may
---     need to make the SAML NameID type derived from the available SCIM
---     data configurable on a per-team basis in the future, to accomodate
---     different legal uses of externalId by different users.
+--   * A mandatory @SAML.UserRef@ is derived from 'Scim.User.externalId'
+--     and the 'idpId' (retrieved via SCIM token).
+--
+-- FUTUREWORK: We may need to make the SAML NameID type derived from the
+-- available SCIM data configurable on a per-team basis in the future, to
+-- accomodate different legal uses of externalId by different users.
+--
+-- __Emails and phone numbers:__ we prohibit emails and phone numbers for now,
+-- because we'd like to ensure that only verified emails and phone numbers end
+-- up in our database, and implementing verification requires design decisions
+-- that we haven't made yet.
```

Comment:
There is an issue here if we want to support email as the SAML subjectID, but we don't allow to store it.  This should at least be mentioned here, and reject it [here with a helpful error message](https://github.com/wireapp/wire-server/blob/b3e9ab57b45a9f62e37d3544284bd18b4ce8b825/services/spar/src/Spar/App.hs#L170-L176), but we can resolve it later.

Reply:
actually, never mind the link i just gave.  scim should know how to create a valid SubjectID, and if that even can be configured, then setting it to "email" is just not an option for now.  so the error happens either in scim setup, or setup doesn't talk about this and there is only one way of computing `UserRef` from the scim schema.

Then we may still want to have a way to turn off implicit user creation.